### DISCUSSION TITLE
How do I read the test suite results?

### DISCUSSION THREAD

**Discussion by J-M0** [newren/git-filter-repo#332] [Q&A] [answered]
I'm trying to package up git-filter-repo for distribution in Cygwin, MSYS2, and Git for Windows and part of that involves running the tests. Some of the tests are failing, but I don't understand how to see which tests failed and why.

**Comment by newren** [comment, marked as answer]
Sorry for the delay.  You can run individual tests in verbose mode, having them quit immediately on first failure, and output trace information as it goes as follows:

$ cd t
$ ./t9390-filter-repo.sh --ver --imm -x

(Or replace t9390-filter-repo.sh with one of the other test scripts.)

However, I suspect the failures you were seeing were likely the issues fixed by commit 838bdd19f6cb (Update expected test data for git 2.35, 2022-03-06).
